config/hodgepodge.cfg

```diff
@@ -322,6 +463,9 @@ tweaks {
     # Stops rendering potion particles from yourself
     B:hidePotionParticlesFromSelf=true
 
+    # give ic2 cells containers like gregtech cells do
+    B:"ic2CellWithContainer "=true
```

Comment:
can you remove the space in the String please

`B:"ic2CellWithContainer"=true`